#!/usr/bin/env python
PACKAGE = "compton_camera_filter"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("tracking_radius", double_t, 1, "orbiting radius", 0.0, 0.0, 100)
gen.add("tracking_speed", double_t, 2, "orbiting speed", 0.0, 0.0, 10)
gen.add("tracking_height", double_t, 3, "orbiting height", 0.0, 0.0, 10)

gen.add("searching_radius", double_t, 4, "orbiting radius", 0.0, 0.0, 100)
gen.add("searching_speed", double_t, 5, "orbiting speed", 0.0, 0.0, 10)
gen.add("searching_height", double_t, 6, "orbiting height", 0.0, 0.0, 10)

exit(gen.generate(PACKAGE, "TrajectoryPlanner", "trajectory_planner"))
